monitor/palma_analyze

```diff
@@ -114,6 +145,10 @@ class PalmaAnalyzer(object):
                             help="print summary")
         parser.add_argument("-n", "--noexit", action="store_true",
                             help="don't exit on error")
+        parser.add_argument("-m", "--monthly",action="store_true", 
+                            help="print monthly summary")
+        parser.add_argument("-mcsv", "--monthlycsv",action="store_true", 
+                            help="output monthly analysis in csv file")                    
```

Comment:
```suggestion
        parser.add_argument("-c", "--csv",action="store_true", 
                            help="output to csv file")                    
```
Short arguments *must* consist of a single character. I suggest using `-c` / `--csv` so that the same flag could be used in a more general context (e.g. for different time spans).